package.json

```diff
@@ -62,5 +62,8 @@
     ],
     "onlyPublishWithReleaseLabel": true
   },
-  "packageManager": "yarn@3.2.3"
+  "packageManager": "yarn@3.2.3",
+  "volta": {
+    "node": "14.21.1"
```

Comment:
Drive by fix: the site doesn’t build on Node 16 currently, which was my default. The TypeScript team is using volta to manage versions in the TypeScript repo.